cmd/goal/account.go

```diff
@@ -539,6 +549,33 @@ var listCmd = &cobra.Command{
 	},
 }
 
+var assetDetailsCmd = &cobra.Command{
```

Comment:
It would be nice to add a very basic test of this command to `test/scripts/e2e_subs/goal-account-asset.sh`